docs/install.rst

```diff
@@ -117,3 +117,73 @@ Troubleshooting
 If you run into any errors after installation, try updating all conda packages to the latest versions using the following command::
 
     conda update --all
+
+
+Frequently asked questions
+--------------------------
+
+**Why are the plots in my notebook not interactive?**
+
+To enable interactive plots, you have to invoke the correct magic commands in the notebook. When using Jupyter notebook,
```

Comment:
Maybe link `magic commands` to this https://ipython.readthedocs.io/en/stable/interactive/magics.html So that readers don't think this is just us being funny :)